src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -38,6 +39,17 @@ class Post911GIBStatusApp extends React.Component {
     // TODO: change the service name below from "user-profile" to
     // something like "post-911-gib-status" once its defined in vets-api
     const { enrollmentState } = this.props;
+
+    const enrollmentHistory = enrollmentState && enrollmentState.enrollmentList.length > 0 ? (
+      enrollmentState.enrollmentList.map((enrollment, index) => {
+        return (
+          <EnrollmentPeriod
+              key={index}
```

Comment:
Apparently you're not supposed to use `index` for the `key`: https://facebook.github.io/react/docs/lists-and-keys.html#keys. But there's not a unique id for each enrollment that I could use instead, so either I just use index or make up a number. Do you have thoughts on what's best here @raquelromano-gov?